src/parser/CodeBlock.h

```diff
@@ -361,6 +361,7 @@ class InterpretedCodeBlock : public CodeBlock {
         bool m_isDuplicated;
         int32_t m_index;
         AtomicString m_name;
+        Node* m_initializer;
```

Comment:
I'm wondering that is it safe to hold an raw address of `Node` element? Since every allocated `Node` is managed by reference count(`RefPtr`), I think that `m_initializer` need to be defined as `RefPtr<Node>`.